### PR TITLE
Make a --quiet flag

### PR DESCRIPTION
**Pull request by rbvigilante**

### CODE REVIEW

File: output/output.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+func Printf(format string, objects ...interface{}) (int, error) {
+	if viper.GetBool("cli.quiet") {
+		return 0, nil
+	}
+
+	return fmt.Printf(format, objects...)
+}
+
+func ErrorPrintln(data interface{}) (int, error) {
+	if viper.GetBool("cli.quiet") {
```
Comment (toothbrush) [inline output/output.go]: I wonder if it's wise to swallow errors, too, when using `--do-not-make-a-noise`?

**Comment by rb-johnmurphy** [comment]
Seems like the role of a logging library, but otherwise nothing inherently wrong.

File: cmd/root.go [hunk 2]
```diff
@@ -60,17 +61,19 @@ func init() {
...
 	viper.BindPFlag("aws.session_duration", rootCmd.PersistentFlags().Lookup("aws-session-duration"))
 	viper.BindPFlag("cache.no_cache", rootCmd.PersistentFlags().Lookup("no-cache"))
 	viper.BindPFlag("cache.cache_only", rootCmd.PersistentFlags().Lookup("cache-only"))
+	viper.BindPFlag("cli.quiet", rootCmd.PersistentFlags().Lookup("quiet"))
 }
 
 func initCache() {
 	dir, err := homedir.Dir()
 
 	if err != nil {
```
Comment (lucaswilric) [inline cmd/root.go]: This block gets repeated *a lot*. Is that OK? Should the behaviour be consolidated in a function?

Comment (rbvigilante) [reply cmd/root.go]: Maybe it should, now that we basically have to do an `os.Exit` rather than returning. I'm not sure.

**Comment by rbvigilante** [comment]
New PR incoming.

**Pull request closed without merging**